<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageRestoration</class>
 <widget class="QWidget" name="ImageRestoration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图像复原</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="originImageLabel">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>请点击下方按钮打开图像</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="5" colspan="4">
    <widget class="QLabel" name="restoredImageLabel">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>请点击下方按钮打开图像</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_1">
     <property name="text">
      <string>长度</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="lengthDoubleSpinBox">
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="value">
      <double>30.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>角度</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QDoubleSpinBox" name="angleDoubleSpinBox">
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
     <property name="value">
      <double>11.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>k</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QDoubleSpinBox" name="kDoubleSpinBox">
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="7" colspan="2">
    <widget class="QPushButton" name="openImageButton">
     <property name="text">
      <string>打开图像</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
